third_party/packages/cupertino_icons/test/icons_list.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+int _compare(IconData a, IconData b) => a.codePoint.compareTo(b.codePoint);
+
+/// The full list of [IconData] constants defined in the [CupertinoIcons] class,
+/// sorted by their code points in ascending order.
+///
+/// This list cannot be mutated.
```

Comment:
This automatically generated. Is there a way to incorporate the code gen step on CI?

Reply:
> This automatically generated.

By what? There should be a prominent comment in this file explain that it is generated, and how to regenerate it.

> Is there a way to incorporate the code gen step on CI?

Since this is included from another file, not currently; there is not currently a mechanism to generate files before `analyze` is run.